cluster/environments/azure-multiple-clusters/README.md

```diff
@@ -15,6 +15,52 @@ is created in.
 
 **This is a placeholder, this repo currently only deploys the second method -- where the Public IP is in the same resource group as the AKS cluster**
 
+Deploying this environment requires a service principal.  To create that service principal, one uses the Azure CLI and issues the following command:
+
+```bash
+> az ad sp create-for-rbac
+{
+  "appId": "4a1dff34-aaaa-bbbb-b62e-dddd5d601a27",
+  "displayName": "azure-cli-2019-03-05-01-59-32",
+  "name": "http://azure-cli-2019-03-05-01-59-32",
+  "password": "5ee46342-eeee-1111-94c1-37c0d6d1cccc",
+  "tenant": "abcd88bf-cccc-41af-91ab-2d7cd0111234"
+}
+```
+
+Now, one uses the above Service Principal for two uses.  First, to set environment variables required
```

Comment:
It would be good to add details that using the same auth service principal, aks sp will have Contributor/Owner role in the entire subscription. Creating a new separate SP for AKS will limit the role of the AKS service principal to the cluster vs. using the authentication Service Principal with Contributor/Owner role on the subscription.

Reply:
This issue will also come up while documenting the "advanced" use case -- anywhere an SP with owner level permissions must be granted to the deploying SP/user.  See issue #198